lib/improver/psychrometric_calculations/psychrometric_calculations.py

```diff
@@ -385,16 +377,11 @@ def _calculate_mixing_ratio(self, temperature, pressure):
         svp = self.pressure_correct_svp(svp, temperature, pressure)
 
         # Calculation
-        result_numer = (consts.EARTH_REPSILON * svp.data)
-        max_pressure_term = np.maximum(svp.data, pressure.data)
+        result_numer = (consts.EARTH_REPSILON * svp)
+        max_pressure_term = np.maximum(svp, pressure)
         result_denom = (max_pressure_term - ((1. - consts.EARTH_REPSILON) *
-                                             svp.data))
-        mixing_ratio = temperature.copy(data=result_numer / result_denom)
-
-        # Tidying up cube
-        mixing_ratio.rename("humidity_mixing_ratio")
-        mixing_ratio.units = Unit("1")
-        return mixing_ratio
+                                             svp))
+        return result_numer / result_denom
 
     def calculate_wet_bulb_temperature(self, temperature, relative_humidity,
```

Comment:
This is still all in cubes, and it says it does the same as _calculate_wbt so do we need both of them?

Reply:
so `calculate_wet_bulb_temperature` has everything in cubes and passes only the numpy arrays to `_calculate_wbt`.
The long term goal of this work is to be able to have the code stay in pure numpy, so the first function does the converting and then from there it does the equations.